src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -424,6 +424,9 @@ def default_api_version(self):
     ResourceType.MGMT_CONTAINERSERVICE: {
         # src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_custom.py:50
         'ManagedClusterAddonProfile': '2020-03-01',
+    },
+    ResourceType.MGMT_MSI: {
+        'user_assigned_identities': '2022-01-31-preview',
```

Comment:
It's recommended to add a comment showing where `2022-01-31-preview` is used, like L425.

In this case, it'll be

```py
        # src/azure-cli/azure/cli/command_modules/identity/_client_factory.py:18
```